packages/server/src/integrations/tests/sqlAlias.spec.ts

```diff
@@ -164,14 +164,23 @@ describe("Captures of real examples", () => {
     it("should handle over 'z' max character alias", () => {
       const tableNames = []
       for (let i = 0; i < 100; i++) {
-        tableNames.push(generator.word())
+        tableNames.push(generator.guid())
       }
       const aliasing = new AliasTables(tableNames)
       let alias: string = ""
       for (let table of tableNames) {
         alias = aliasing.getAlias(table)
       }
-      expect(alias).toEqual("aaay")
+      expect(alias).toEqual("cv")
+    })
+  })
+
+  describe("check some edge cases", () => {
+    it("should handle table names/columns with dots in them", () => {
+      const tableNames = ["hello.world", "foo.bar.baz"]
+      const aliasing = new AliasTables(tableNames)
+      const aliased = aliasing.aliasField("hello.world.field")
```

Comment:
In the SQL, the field would never appear like this. It would be `"hello.world"."field"` or some such. I'd modify this test case to be like that, and also have one that uses grave accents instead of double quotes.